### PR TITLE
remove ruby build altogether #3035

### PR DESCRIPTION
**Pull request by finnfiddle**
#### Before the PR (what is the issue or what needed to be done)
Now that we are using webpack for the static assets build - we dont need ruby anymore.
#### The solution
Remove ruby and all deps.
## Checklist
* [ ] Connect the issue
* [ ] Test plan
* [ ] Copyright header
* [ ] Code formatting
* [ ] Documentation

### CODE REVIEW

File: ci/devserver.sh [hunk 1]
```diff
@@ -22,9 +22,8 @@ SRC_DIR="/app/src"
 
 cd "$SRC_DIR/Dashboard"
 
-bundle exec rake watchbg > "$SRC_DIR/rake.log" 2>&1 &
-bundle exec npm install &
-bundle exec npm run build:dev > "$SRC_DIR/build.dev.log" 2>&1 &
+npm install &
+npm run build:dev > "$SRC_DIR/build.dev.log" 2>&1 &
```
Comment (dlebrero) [inline ci/devserver.sh]: Do we want to run `npm install` and `npm run` in parallel? I understand that this is not new behavior but I am surprised nonetheless

File: Dashboard/scripts/rebuild_dev_env_macos.sh [hunk 2]
```diff
@@ -57,7 +57,6 @@ function ensure_build_tools_are_installed
     brew update
```
Comment (dlebrero) [inline Dashboard/scripts/rebuild_dev_env_macos.sh]: AFAIK, we can delete the whole file as any dev setup should be done by Docker. @stellanl ?

**Comment by dlebrero** [comment]
There is another Ruby file at https://github.com/akvo/akvo-flow/blob/20152ca7ce49033331ede4be3c6fddc42f4380e4/Dashboard/config/application.rb. Maybe it can also be removed

**Comment by stellanl** [comment]
Building this branch on my machine (which has npm 3.5.2 installed)  after clearing the target directory I get a number of npm warnings, but no error for /app/src/Dashboard:
```
akvo-flow_1_dbc781e50605 | Compiling "../../../GAE/target/akvo-flow/admin/frames/users.js" from ["src"]...
akvo-flow_1_dbc781e50605 | npm WARN @babel/plugin-syntax-object-rest-spread@7.2.0 requires a peer of @babel/core@^7.0.0-0 but none is installed. You must install peer dependencies yourself.
akvo-flow_1_dbc781e50605 | npm WARN babel-jest@24.1.0 requires a peer of @babel/core@^7.0.0 but none is installed. You must install peer dependencies yourself.
akvo-flow_1_dbc781e50605 | npm WARN handlebars-loader@1.7.1 requires a peer of handlebars@>= 1.3.0 < 5 but none is installed. You must install peer dependencies yourself.
akvo-flow_1_dbc781e50605 | npm WARN akvo-flow@ No repository field.
akvo-flow_1_dbc781e50605 | npm WARN akvo-flow@ No license field.
akvo-flow_1_dbc781e50605 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.7 (node_modules/fsevents):
akvo-flow_1_dbc781e50605 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
akvo-flow_1_dbc781e50605 | 
akvo-flow_1_dbc781e50605 | added 67 packages in 11.803s

```

**Comment by finnfiddle** [comment]
@dlebrero ok should be resolved now everything

File: ci/devserver.sh (hunk 1 above)
Comment (dlebrero) [reply ci/devserver.sh]: How is this resolved? It is causing the issue described above.

Comment (finnfiddle) [reply ci/devserver.sh]: isnt it fixed now @dlebrero

Comment (dlebrero) [reply ci/devserver.sh]: It is not. Running `npm install` and `npm run` at the same time is causing the above issue.

Comment (finnfiddle) [reply ci/devserver.sh]: does this not run them sequentially? ```npm install
npm run build:dev > "$SRC_DIR/build.dev.log" 2>&1 &```

File: ci/devserver.sh [hunk 3]
```diff
@@ -1,3 +1,4 @@
+b
```
Comment (finnfiddle) [inline ci/devserver.sh]: @dlebrero is this meant to be here?

Comment (dlebrero) [reply ci/devserver.sh]: No.

**Pull request merged**